packages/react-native/Libraries/NativeComponent/BaseViewConfig.android.js

```diff
@@ -164,6 +164,9 @@ const directEventTypes = {
 const validAttributesForNonEventProps = {
   // @ReactProps from BaseViewManager
   backgroundColor: {process: require('../StyleSheet/processColor').default},
+  experimental_backgroundImage: {
+    process: require('../StyleSheet/processBackgroundImage').default,
+  },
```

Comment:
Related to comment below about keeping this off of BaseViewManager, if we start with just `<View>`, could we keep local to that view manager and viewconfig?